src/internal/m365/sharepoint/backup_test.go

```diff
@@ -174,11 +175,14 @@ func TestSharePointPagesSuite(t *testing.T) {
 	suite.Run(t, &SharePointPagesSuite{
 		Suite: tester.NewIntegrationSuite(
 			t,
-			[][]string{tester.M365AcctCredEnvs},
-		),
+			[][]string{tester.M365AcctCredEnvs}),
 	})
 }
 
+func (suite *SharePointPagesSuite) SetupSuite() {
+	graph.InitializeConcurrencyLimiter(4)
```

Comment:
What's the reason behind specifying limiter initialization for various tests? It feels like a bad pattern. 

Also, side note: We need to disable limiter for sharepoint.

Reply:
nvm, looks like you are testing whether we turned off the limiter for sharepoint/OD?
A comment would help add some context.

Reply:
> What's the reason behind specifying limiter initialization for various tests? It feels like a bad pattern & can be accidentally missed for newer tests.

If we don't initialize it, the test will error  out.  When I have some time, I'm going to standardize around a test setup factory [like this](https://github.com/alcionai/corso/blob/main/src/pkg/services/m365/api/helper_test.go#L85), and we'll be able to include it there easily.  In the meantime, I'm just going to hack in the calls.

> Also, side note: We need to disable limiter for sharepoint.

In tests?  I don't think that's a need so much as a want.